### PR TITLE
add driver for Oxford Instruments ILM 200 Helium Level Meter

### PR DESCRIPTION
**Pull request by peendebak**
@jenshnielsen

### CODE REVIEW

File: qcodes/instrument_drivers/oxford/ILM200.py [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+        # because protocol has no termination chars the read reads the number
+        # of bytes in the buffer
+        bytes_in_buffer = self.visa_handle.bytes_in_buffer
+        # a workaround for a timeout error in the pyvsia read_raw() function
+        with(self.visa_handle.ignore_warning(visa.constants.VI_SUCCESS_MAX_CNT)):
+            mes = self.visa_handle.visalib.read(
+                self.visa_handle.session, bytes_in_buffer)
+        # cannot be done on same line for some reason
+        mes = str(mes[0].decode())
+        # if mes[1] != 0:
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/oxford/ILM200.py]: delete ?

Comment (jenshnielsen) [inline qcodes/instrument_drivers/oxford/ILM200.py]: delete ?

File: qcodes/instrument_drivers/oxford/ILM200.py [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+            0 : "SLOW"
+            1 : "FAST"
+        """
+        self.set_remote_status(1)
+        if rate == 0:
+            self.set_to_slow()
+        elif rate == 1:
+            self.set_to_fast()
+        self.set_remote_status(3)
+        print(self._do_get_rate())
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/oxford/ILM200.py]: Change to log?

File: qcodes/instrument_drivers/oxford/ILM200.py [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+        return float(result.replace("R", "")) / 10
+
+    def _do_get_status(self):
+        """
+        Get status of the device.
+        Input:
+            None
+
+        Output:
+            None
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/oxford/ILM200.py]: this returns the status?

File: qcodes/instrument_drivers/oxford/ILM200.py [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+        # current_flowing = {
+        # 0 : "Curent not flowing in Helium Probe Wire",
+        # 1 : "Curent not flowing in Helium Probe Wire"
+        # }
+        # auto_fill_status = {
+        # 00 : "End Fill (Level > FULL)",
+        # 01 : "Not Filling (Level < FULL, Level > FILL)",
+        # 10 : "Filling (Level < FULL, Level > FILL)",
+        # 11 : "Start Filling (Level < FILL)"
+        # }
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/oxford/ILM200.py]: Any reason this is commented out and not implemented?

Comment (peendebak) [inline qcodes/instrument_drivers/oxford/ILM200.py]: @jenshnielsen Not sure why this is here. Probably left for implementation later on.

**Pull request merged**